app/views/layouts/application.html.erb

```diff
@@ -53,7 +53,7 @@
     <% end %>
     <!-- Site Wide Scipts -->
     <script>
-      $(document).on('turbolinks:load', AppLayout.initialize(<%= user_signed_in? %>, <%= current_user ? current_user.id : nil %>));
```

Comment:
This is where that initialized function is being called but it was just failing??? and logging a console error